src/containers/GraphSettingsContainer.tsx

```diff
@@ -75,8 +78,12 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps> {
         ...this.getGraphParams(),
         injectServiceNodes: this.props.showServiceNodes
       });
-    } else if (this.props.showSecurity && this.props.showSecurity !== prevProps.showSecurity) {
-      // when turning on security we need to perform a fetch, because we don't pull security data by default
+    } else if (
+      (this.props.showSecurity && this.props.showSecurity !== prevProps.showSecurity) ||
+      this.props.showUnusedNodes !== prevProps.showUnusedNodes
```

Comment:
why does the first part check for truthiness of showSecurity but you do not do this for showUnusedNodes?

I see "this.props.showSecurity && ..." but I do not see "this.props.showUnusedNodes && ..."

Reply:
It's not a mistake :)  For show security we only need to fetch if the option gets turned on.  When turned off we simply don't show the badging (in graphstyles).  But unused nodes are part of the total topology.  We want to fetch any time it is toggled.